src/Resources/public/js/fieldConfigDialog.js

```diff
@@ -54,13 +55,20 @@ pimcore.plugin.datahub.fieldConfigDialog = Class.create({
         buttons = [];
 
         buttons.push({
-                text: t("apply"),
-                iconCls: "pimcore_icon_apply",
-                handler: function () {
-                    this.commitData();
-                }.bind(this)
-            }
-        );
+            text: t("plugin_pimcore_datahub_add_all_definitions"),
```

Comment:
The `plugin_pimcore_datahub_add_all_definitions` translation is missing.
![image](https://github.com/user-attachments/assets/2f90292e-3541-4a9d-aca1-fc373a3fe12b)

Reply:
Done.